native/ios/Integreat/Info.plist

```diff
@@ -32,8 +32,6 @@
 	<true/>
 	<key>NSAppTransportSecurity</key>
 	<dict>
-		<key>NSAllowsArbitraryLoads</key>
-		<true/>
```

Comment:
I think we need this for HTTP traffic: https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads

Reply:
Hmm, react native upgrade helper says to remove it: https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.2 @maxammann

Reply:
Maybe RN had it previously by default. But I think we really need it. Unless there is an other option in XCode to allow HTTP traffic. By default it is forbidden on iOS and Android.

Reply:
HTTP is not working on Android either I think right now